Update rendering-scripts.md for clarity on validation scripts (#8104)
umbraco/UmbracoDocs: 17/umbraco-forms/developer/rendering-scripts.md

When setting up templates for Umbraco Forms, two separate script-rendering methods are involved, and both are required for forms to work correctly.

> `@Html.RenderUmbracoFormDependencies(Url)`, covered in the [Preparing Your Frontend](../../../umbraco-forms/developer/prepping-frontend/) article, renders the ASP.NET Client Validation library, which Umbraco Forms uses by default for client-side validation. This helper goes in the `<head>` of your template. If your project uses jQuery validation instead, omit this call and reference dependencies manually, as described in the [Preparing Your Frontend](../../../umbraco-forms/developer/prepping-frontend/) article.

Reviewer: ```suggestion
`@Html.RenderUmbracoFormDependencies(Url)`, covered in the [Preparing Your Frontend](prepping-frontend.md) article, renders the ASP.NET Client Validation library, which Umbraco Forms uses by default for client-side validation. This helper goes in the `<head>` of your template. If your project uses jQuery validation instead, omit this call and reference dependencies manually, as described in the [Preparing Your Frontend](prepping-frontend.md) article.
```

> `@Html.RenderUmbracoFormDependencies(Url)`, covered in the [Preparing Your Frontend](../../../umbraco-forms/developer/prepping-frontend/) article, renders the ASP.NET Client Validation library, which Umbraco Forms uses by default for client-side validation. This goes in the `<head>` of your template. If you have configured the project to use jQuery validation instead, this call should be omitted and the jQuery dependencies should be referenced manually as described in the [Preparing Your Frontend](../../../umbraco-forms/developer/prepping-frontend/) article.

Reviewer: ```suggestion
`@Html.RenderUmbracoFormDependencies(Url)`, covered in the [Preparing Your Frontend](../../../umbraco-forms/developer/prepping-frontend/) article, renders the ASP.NET Client Validation library, which Umbraco Forms uses by default for client-side validation. This helper goes in the `<head>` of your template. If your project uses jQuery validation instead, omit this call and reference dependencies manually, as described in the [Preparing Your Frontend](../../../umbraco-forms/developer/prepping-frontend/) article.
```
